drf_braces/tests/test_utils.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import print_function, unicode_literals
+
+import unittest
+
+from drf_braces.utils import (
+    get_class_name_with_new_suffix,
+    get_attr_from_base_classes,
+)
+from rest_framework import fields
+
+
+class TestUtils(unittest.TestCase):
+    def test_get_class_name_with_new_suffix(self):
+        new_name = get_class_name_with_new_suffix(
+            klass=fields.IntegerField,
+            existing_suffix='Field',
+            new_suffix='StrawberryFields'
+        )
+        self.assertEqual(new_name, 'IntegerStrawberryFields')
+
+        new_name = get_class_name_with_new_suffix(
+            klass=fields.IntegerField,
+            existing_suffix='straws',
+            new_suffix='Blueberries'
+        )
+        self.assertEqual(new_name, 'IntegerFieldBlueberries')
+
+    def test_get_attr_from_base_classes(self):
+        Parent = type(str('Parent'), (), {'fields': 'pancakes'})
+
+        self.assertEqual(
+            get_attr_from_base_classes((Parent,), [], 'fields'), 'pancakes'
+        )
+
+        self.assertEqual(
+            get_attr_from_base_classes(
+                (Parent,), {'fields': 'mushrooms'}, 'fields'
+            ),
+            'mushrooms'
+        )
+
+        self.assertEqual(
+            get_attr_from_base_classes((Parent,), [], '', default='maple_syrup'),
```

Comment:
Is `default` used somewhere? I saw that `default` is provided in the code as `FormSerializerOptions`. This seems odd to me since we are returning a string or a class.

Refer especially to the [docstring](https://github.com/dealertrack/django-rest-framework-braces/pull/3/files#diff-3480c252ba6497a8f23681716ab1bc8cR154)